server/web/code/src/composeexample/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for composeexample project.
```

Comment:
Right, so why are you commiting my example?
Shouldn't you create the Django server for superpac?